x-pack/test/functional/apps/saved_query_management/feature_controls/security.ts

```diff
@@ -10,7 +10,14 @@ import { getSavedQuerySecurityUtils } from '../utils/saved_query_security';
 
 type AppName = 'discover' | 'dashboard' | 'maps' | 'visualize';
 
-const apps: AppName[] = ['discover', 'dashboard', 'maps', 'visualize'];
+const apps: AppName[] = [
+  'discover',
+  'dashboard',
+  // Commenting out maps to due to test flakiness, re-enable to resolve
+  // https://github.com/elastic/kibana/issues/183066
+  // 'maps',
+  'visualize',
+];
```

Comment:
The maps test failures seem to stem from a transient service interruption. We should be able to include the maps tests again.